<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-BMK3LDR" failures="0" tests="3" name="TestNg.timeOutkeyword" time="2.015" errors="1" timestamp="2025-10-14T21:35:26 IST" skipped="1">
  <testcase classname="TestNg.timeOutkeyword" name="TC2" time="2.013">
    <error message="Method TestNg.timeOutkeyword.TC2() didn&#039;t finish within the time-out 2000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method TestNg.timeOutkeyword.TC2() didn't finish within the time-out 2000
at java.base/java.lang.Thread.sleep0(Native Method)
at java.base/java.lang.Thread.sleep(Thread.java:509)
at TestNg.timeOutkeyword.TC2(timeOutkeyword.java:10)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </error>
  </testcase> <!-- TC2 -->
  <testcase classname="TestNg.timeOutkeyword" name="show" time="0.002"/>
  <testcase classname="TestNg.timeOutkeyword" name="test" time="0.000">
    <skipped/>
  </testcase> <!-- test -->
  <system-out>
    <![CDATA[div
mul
sub
before class]]>
  </system-out>
</testsuite> <!-- TestNg.timeOutkeyword -->
